tools/twix/src/twix_painter.rs

```diff
@@ -382,6 +382,18 @@ impl<Frame> TwixPainter<Frame> {
             stroke,
         );
     }
+
+    pub fn text(
+        &self,
+        position: Point2<Frame>,
+        align: eframe::emath::Align2,
+        text: String,
+        font_id: eframe::epaint::FontId,
+        color: Color32,
+    ) {
+        let position = self.transform_world_to_pixel(position);
+        self.painter.text(position, align, text, font_id, color);
```

Comment:
Interesting, this shape doesn't rotate and scale with the transform...
I guess that's a good thing for the kind of floating label text you want, but it wouldn't be fit to render text into the world.
Can you find a name for this function which expresses this property?